lib/ddr/ingesttools/dpc_folder_converter/converter.rb

```diff
@@ -66,6 +67,17 @@ def find_component_files(dir)
         files
       end
 
+      def find_intermediate_files(dir)
+        files = []
+        Find.find(dir) do |path|
+          next unless path.include?(INTERMEDIATE_FILES_DIRNAME)
```

Comment:
@dchandekstark I debated about that and decided to treat the occurrence of a folder by that name anywhere in the source tree as representing intermediate files.  I use the same logic for finding 'targets' directories.